snippets/nextYear.md

```diff
@@ -0,0 +1,28 @@
+---
+title: nextYear
+tags: date,intermediate
+---
+
+Returns this same day, next year, while accounting for leap years.
```

Comment:
Ask a user to enter a whole number	 
  
 
	 
  
Returns this same day, next year, while accounting for leap years.	 
  
 
	 
  
 - Use the parameter retrieves the desired date or use `new Date()` to get the current date.	 
  
 - Use `Date.prototype.getYear()` to retrieve the year and decide which year we should use for February check.	 
  
 - Use `Date.prototype.getMonth()` to check if next february is this year or next.	 
  
 - Use `Date.prototype.getMonth()` to check if next February is 29 days or not (and adjust the number of days accordingly).	 
  
- Use `Date.prototype.setDate()` to set the day one year ahead.